tests/core/FS_exports_assert.out

```diff
@@ -1 +1 @@
-You can force-include filesystem support with  -s FORCE_FILESYSTEM=1
+You can force-include filesystem support with  -s FORCE_FILESYSTEM=1)
```

Comment:
This paren is not balanced?

Reply:
Its odd itsn't.  Without this the tests fail.. this is the end of one of the output lines and it appears the matcher wants the end of the line to be complete but is OK with head being missing.  Will investigate further.

Reply:
Oh, its because the newline at the end of the line here is part of the pattern we are logging for.    If the line doesn't end in `=1\n` then it won't match.

Reply:
Hmm, not sure I follow...

What is emitting the `)`? Can we emit a `(` to balance it?

Reply:
Both before and after the change what we are checking for here is the tail end of much longer error message.  To make this balanced I would have to include the full error message:

https://github.com/emscripten-core/emscripten/blob/aca7c128fd844d2b2b306aa2d5205a07f053bbe0/src/preamble.js#L631

Would you prefer that I include the entire message here?

Reply:
Oh, now I see. Yes, maybe let's include the full text here? Then it's testing more, and it's less surprising.

Reply:
I don't think I see that update in the PR here? It is still just `You can force-include filesystem support with -s FORCE_FILESYSTEM=1)`